lib/atlas/scaler/graph_persistor.rb

```diff
@@ -0,0 +1,26 @@
+module Atlas
+  class Scaler::GraphPersistor
+    def initialize(dataset, path)
+      @dataset = dataset
+      @graph   = GraphBuilder.build
+      @path    = path
+    end
+
+    def persist_graph
```

Comment:
> I would rather change it to persist! since than it's actually clear to me.

Agreed, but my (general) rule is that if a class name is self-describing, and has only one useful method (`persist_graph`), you can opt for a generically-named *class method* to simplify the whole API. Since the class *name* describes what it does, the method name is less important:

```ruby
# Use
GraphPersistor.persist(graph, path)
GraphPersistor.run(graph, path)
GraphPersistor.call(graph, path)
...

# Implementation
class GraphPersistor
  def self.persist(graph, path)
    new(graph).save_to(path)
  end
end
```

I'll often go a step further and name the class method `call` since this allows you to transparently substitute the implementation with a `lambda {}`. It's also quite nice [if you have multiple such "processes" that you need to run](https://github.com/quintel/refinery/tree/master/lib/refinery/catalyst) (e.g. Scaler -> Exporter -> Persistor). A common API among such classes/lambdas is quite clean and easy-to-use. I'll leave that up to your judgement though.